### PR TITLE
Add reduction parameter to mean_pairwise_squared_error loss

### PR DESCRIPTION
**Pull request by chrisyeh96**
- add reduction parameter to `tf.losses.mean_pairwise_squared_error` to make it consistent with all of the other loss functions
- increased clarify of the documentation for the function
- use the `axis` parameter instead of `reduction_indices` for `math_ops.reduce_sum()` because `reduction_indices` is deprecated

### CODE REVIEW

**Comment by jhseu** [comment]
Actually, would you mind redoing this on top of master? We're not taking changes to the 1.6 branch. Closing this out, but please send another PR.
